File: min-max-sum.js
function miniMaxSum(arr) {
    // Write your code here
    let sortedArray = arr.sort((a, b) => a - b)
    let initialSum = 0
    let minSum = sortedArray.slice(0, 4).reduce(
          (accumulator, currentValue) => accumulator + currentValue,
  initialSum,
    )
    let maxSum = sortedArray.slice(1, 5).reduce(
          (accumulator, currentValue) => accumulator + currentValue,
  initialSum,
    )
    
    console.log(`${minSum} ${maxSum}`)
    
}

File: min-max-sum.py
def miniMaxSum(arr):
    
    def add(x, y):
        return x + y
    
    sorted_values = sorted(arr)
    min = reduce(add, sorted_values[0:4])
    max = reduce(add, sorted_values[1:5])
    
    print(min, max)
